examples/00-fluent/modeling_cavitation.py

```diff
@@ -0,0 +1,377 @@
+"""
+
+Modeling Cavitation
+-------------------
+This example examines the pressure-driven cavitating flow of water
```

Comment:
If you are editing this stuff in Visual Studio Code you should install this extension:

https://marketplace.visualstudio.com/items?itemName=stkb.rewrap

and rewrap to 80 columns.